app/src/main/java/org/p2p/wallet/jupiter/ui/main/JupiterSwapPresenter.kt

```diff
@@ -70,7 +70,8 @@ class JupiterSwapPresenter(
     private val rateTickerManager: SwapRateTickerManager,
     private val dispatchers: CoroutineDispatchers,
     private val userLocalRepository: UserLocalRepository,
-    private val historyInteractor: HistoryInteractor
+    private val historyInteractor: HistoryInteractor,
+    private val initialAmount: String? = "0",
```

Comment:
how it is null, but has default parameter?

Reply:
you're right, it should remain null